docs/dictionary/property/icon.lcdoc

```diff
@@ -46,23 +46,20 @@ the <OS X> dock.
 
 LiveCode looks for the specified image in the following order:
 
-1) The stack of the object's <behavior> (if applicable)
-2) The stack of the owner of the object's <behavior> (if applicable)
-
-    ...
-
-n) The stack of the object's stack's <behavior> (if applicable)
-A) The object's stack
-B) The object's stack's mainstack (if a substack)
-C) The object's stack's mainstacks substacks
-D) The list of open stacks, in order they were loaded
+1. The stack of the object's <behavior> (if applicable)
+2. The stack of the owner of the object's <behavior> (if applicable)
+3. The stack of the object's stack's <behavior> (if applicable)
+4. The object's stack
+5. The object's stack's mainstack (if a substack)
+6. The object's stack's mainstack's substacks
+7. The list of open stacks, in order they were loaded
```

Comment:
This slightly loses the sense of the original's iteration up the owner chain before you get to the object's stack's behavior... mind you, I'm not really sure how to render that in markdown. It might work to use an unordered list instead, and prefix with the original labels, eg
```
* 1) The stack of the object's <behavior> (if applicable)
* 2) The stack of the owner of the object's <behavior> (if applicable)
*      ...
* n) The stack of the object's stack's <behavior> (if applicable)
* A) The object's stack
``` 
etc